src/schema/slack/functions/mod.ts

```diff
@@ -21,10 +21,10 @@ const SlackFunctions = {
   CreateUsergroup: CreateUsergroup,
   Delay: Delay,
   InviteUserToChannel: InviteUserToChannel,
+  OpenForm: OpenForm,
   RemoveUserFromUsergroup: RemoveUserFromUsergroup,
   SendDm: SendDm,
   SendEphemeralMessage: SendEphemeralMessage,
-  SendForm: SendForm,
```

Comment:
We need to manually delete the `SendForm` file, we have a ticket to update the generation script to do this